v1-scripts/generate-build-branch

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+ORIGIN_REMOTE_NAME="${ORIGIN_REMOTE_NAME:-origin}"
+MIRROR_BRANCH="${MIRROR_BRANCH:-bugfix-2.0.x}"
+MACHINES_BRANCH="${MACHINES_BRANCH:-HEAD}"
+BUILD_BRANCH="${BUILD_BRANCH:-v1-$MIRROR_BRANCH}"
+ADD_BRANCHES="${ADD_BRANCHES:-v1-base-config}"
```

Comment:
This v1-base-config has to always be based on a common parent, or it will pull inextra stuff, right? I still have this idea in my head that bugfix and releases are not organized well upstream. Not sure how to fix that though.

Reply:
What if v1-base-config was a submodule (or even just a nested checkout during build).

Reply:
The point of these branches is to carry code that needs to be merged in but upstream will not carry. Think about backports of our PR-s that make it work/build on 2.0.x. These are present in bugfix-2.0.x and will be eventually in 2.0.1 but until that happens, backports need to be stored somewhere.

The v1-base-config branch right now is a test of two concepts: Starting with a "base" config that we (hopefully) pushed to upsteam and this branch merging logic.